src/payment/build.gradle

```diff
@@ -0,0 +1,54 @@
+plugins {
+    id("com.github.johnrengelman.shadow") version "6.1.0"
+    id("io.micronaut.application") version "1.4.2"
+}
+
+version = "0.1"
+group = "payment"
```

Comment:
we could agree on common group name for these service. I have used `group = "micronaut.mushop"` where with `name="carts"` it's more "real-world" scenario. But I don't mind using anything else, just for the sake of integrity ...

Reply:
same applies to package tree .. `payment` vs `mushop.payment` ?

Reply:
Seems reasonable to use `micronaut.mushop`